src/components/PostComment.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react"
+import { postComment } from "../api"
+
+export const PostComment = ({comments, setComments, review_id, user}) => {
+
+    const [showCommentBox, setShowCommentBox] = useState(false)
+    const [commentText, setCommentText] = useState("")
+    const [isCommentPosting, setIsCommentPosting] = useState(false)
+    const [commentPostError, setCommentPostError] = useState(false)
+
+    const handleCommentPost = (e) => {
+        e.preventDefault()
+        setIsCommentPosting(true)  
+        postComment(review_id, user, commentText)
+        .then((newComment) => {
+            setIsCommentPosting(false)
+            setShowCommentBox(false)
+            setComments([newComment, ...comments])
+        })
+        .catch((error) => {
+            setIsCommentPosting(false)
+            setCommentPostError(true)
+        })
+    }
+
+    return(
+        <div className="post-comment-container">
+            <button className="post-comment-button" onClick={() => setShowCommentBox(true)}>Post A Comment</button>
+            {showCommentBox === true ?
+                <form className="post-comment-form" onSubmit={handleCommentPost}>
+                    <label htmlFor="comment-body" >Enter Comment: </label><br></br>
+                    <input className="post-comment-form" type="text" value={commentText} id="comment-body" required onChange={(e) => setCommentText(e.target.value)}></input><br></br>
```

Comment:
Good use of the required attribute here, but be aware that a user can still post an empty space as a comment. You may wish to add further validation later.